client/scss/components/_dropdown.legacy.scss

```diff
@@ -33,9 +33,9 @@
     position: absolute;
     overflow: hidden;
     top: 100%;
-    inset-inline-start: -2000px;
     z-index: 500;
     opacity: 0;
+    pointer-events: none;
```

Comment:
Would we be better using `visibility: hidden;` / `visibility: visible;`? It achieves the same goal and avoids some of the accessibility pitfalls (i.e. should not be 'tabbable'.).

Reply:
We thought about doing it that way, the only thing we noticed was that the dropdown no longer faded out when closed, it just disappears instead.

Also I am a little suprised, I thought we would want the items to remain tabbable, not the opposite. Maybe that should be a seperate issue then as this PR should not affect tabbing behaviour. `pointer-events: none` only affect/disables mouse and touch events. Focus events should remain unaffected by this.

Reply:
Ohh yep. This may need to use CSS animation. 

Maybe it's fine as is (pointer events), the plan is to remove this legacy dropdown in a near future release.

Reply:
It should be fine, at least until you remove the legacy dropdown.

It was kind of funny finding this bug. With the offending code going back at least 9 years to before the wagtail project itself started, to verdant (judging from the git history). It was amazing to find that there were a related issue only a mere week or so old. Quite amazing for such an old "bug". Though thinking about it, maybe there are other issues made over the years too. Never checked.